src/server/app/admin/controllers/users/users.password.server.controller.js

```diff
@@ -219,9 +196,13 @@ exports.reset = (req, res, next) => {
 				html: emailHTML
 			};
 
-			smtpTransport.sendMail(mailOptions, (error) => {
-				done(error, 'done');
-			});
+			emailService.sendMail(mailOptions)
+				.then((result) => {
+					done(null);
+
+				}, (error) => {
+					logger.error({err: error, req: req}, 'Failure sending email.');
```

Comment:
Do we need a `done(error);` here to make sure that the promise is resolved/rejected, or is that unnecessary?